fixtures/input.txt

```diff
@@ -3,7 +3,7 @@ To: foo@example.com, bar@example.com, baz@example.com, quz@example.com, foobar@e
 	foobuzz@example.com, fooquz@example.com, bonk@example.com, bink@example.com, bank@example.com, bossman@example.com
 
 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-
+ 
```

Comment:
Yeah, sorry. There are a few things here:

1. `eol` is horribly named at this point. I should probably rename it to `emptyLine` and use `endOfLine` when that's what I mean.
2. the `whitespaceOnlyLine` should actually capture the `endOfLine` as well. It's kinda weird that it doesn't since `eol` _does_ capture that character.
3. Even though these parsers are capturing these characters, we're ignoring them at the call site with `void`. This makes the most sense to me since it would allow me to capture them if I actually wanted to for some reason.